### PR TITLE
Ci tokenomics emulation

### PR DESCRIPTION
**Pull request by augustgerro**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,3 +7,5 @@ scripts/validator/node_modules
 scripts/generate-accounts/node_modules
 scripts/tokenomics-emulation/node_modules
 .idea
+accounts
+spec-data
```
Comment (AndreiNavoichyk) [inline .gitignore:11]: spec-data should be removed from here

File: docker-compose.yaml [hunk 2]
```diff
@@ -54,6 +57,8 @@ services:
     volumes:
       - ./spec-data:/cereio/spec-data
       - ./chain-data:/data
+    env_file:
+      - ./configs/.env.testnet
```
Comment (AndreiNavoichyk) [inline docker-compose.yaml:61]: This parameter should be passed outside

File: docker-compose.yaml [hunk 3]
```diff
@@ -68,6 +73,8 @@ services:
       - "9933:9933"
       - "30333:30333"
       - "9615:9615"
+    env_file:
```
Comment (AndreiNavoichyk) [inline docker-compose.yaml:76]: This parameter should be passed outside

File: scripts/tokenomics-emulation/generate-chain-spec.sh [hunk 4]
```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env bash
 
-rm -rf spec-data/customSpec.json
-rm -rf spec-data/customSpecRaw.json
+docker-compose down -t 0
```
Comment (AndreiNavoichyk) [inline scripts/tokenomics-emulation/generate-chain-spec.sh:3]: Why `0`?

File: scripts/tokenomics-emulation/generate-chain-spec.sh [hunk 5]
```diff
@@ -1,11 +1,12 @@
...
+rm -rf chain-data accounts spec-data
```
Comment (AndreiNavoichyk) [inline scripts/tokenomics-emulation/generate-chain-spec.sh:4]: Add `scripts/keys`

File: scripts/tokenomics-emulation/generate-chain-spec.sh (hunk 4 above)
Comment (augustgerro) [reply scripts/tokenomics-emulation/generate-chain-spec.sh:3]: It's removes old containers in 10 times faster

File: configs/.env.testnet [hunk 6]
```diff
@@ -1,4 +1,4 @@
-BOOT_NODE_IP_ADDRESS=161.35.229.40
-NETWORK_IDENTIFIER=12D3KooWPwNX9L3oSUmKL3iHJsH9AJeiaEuBqPvajDbH5SZwQLQP
-LOG_LEVEL=debug
+BOOT_NODE_IP_ADDRESS=157.230.113.121
+NETWORK_IDENTIFIER=12D3KooWMh72LWpGZhUWtgPx6Yg4KG4bnBnTZXtg2eWA69n6ZgU6
+LOG_LEVEL=info
```
Comment (AndreiNavoichyk) [inline configs/.env.testnet:3]: Move back to `debug`

File: docker-compose.yaml [hunk 7]
```diff
@@ -29,12 +29,15 @@ services:
   create_chain_spec:
     container_name: create_chain_spec
     image: cerebellumnetwork/pos-node:2.1.0-rc1
+    user: root
```
Comment (AndreiNavoichyk) [inline docker-compose.yaml:32]: Why `root`?

File: scripts/tokenomics-emulation/generate-chain-spec.sh (hunk 4 above)
Comment (AndreiNavoichyk) [reply scripts/tokenomics-emulation/generate-chain-spec.sh:3]: Ok, here we don't have any post-termination actions. So it is safe.

File: docker-compose.yaml (hunk 7 above)
Comment (augustgerro) [reply docker-compose.yaml:32]: Because without these permissions the account generation does not work

**Review by AndreiNavoichyk — COMMENTED** [review]
Revert changes in (.gitkeep, remove submodule, revert .env.testnet, customSpec, customSpecRaw).

File: docker-compose.yaml (hunk 7 above)
Comment (augustgerro) [reply docker-compose.yaml:32]: Also it is a one-time operation similar to how sudo is used to access a folder at one time.  

It is safer than changing permissions on all directories.

And these containers don't work all the time but only run once.  Boot_node and validators does not start from root

**Review by AndreiNavoichyk — APPROVED** [review]

**Pull request merged**